scripts/cmssetup.sh

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Don't change these for the existing scripts. They may be using bash specific commands and changing this could break the script.  All new scripts should be written and tested using `sh` though.